Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json

```diff
@@ -0,0 +1,563 @@
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "2019-10-16"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Nirmala Ramnarine",
+                "orcid": "https://orcid.org/0000-0002-9001-8153",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "nramnarine13@gwu.edu"
+            },
+            {
+                "name": "David T. Ting",
+                "orcid": "https://orcid.org/0000-0002-3261-2322",
+                "affiliation": "Massachusetts General Hospital Cancer Center, MASSACHUSETTS GENERAL HOSPITAL CANCER CENTER, Dana-Farber/Harvard Cancer Center, Massachusetts General Hospital, Harvard-MIT Division of Health Sciences and Technology, Massachusetts Institute of Technology",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": " Ben S. Wittner",
+                "orcid": "https://orcid.org/0000-0001-5657-9849",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Matteo Ligorio",
+                "orcid": "https://orcid.org/0000-0001-8865-0881",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA; Department of Health Sciences, University of Genoa, 16126 Genoa, Italy.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": " Nicole Vincent Jordan",
+                "orcid": "https://orcid.org/0000-0002-0137-5782",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112